package.json

```diff
@@ -26,8 +26,9 @@
     "jest-runner-eslint": "^0.3.0"
   },
   "scripts": {
-    "lint": "prettier src/**/*.js --list-different",
-    "test": "jest"
+    "lint": "jest -c jest.eslint.config.js",
+    "test": "jest",
+    "prettier": "prettier --plugin=. --parser=python"
```

Comment:
Useful for testing. `yarn prettier test.py` or `echo 'foo' | yarn prettier`

Reply:
shouldn't need the explicit `--parser`?

Reply:
Not always, but `--debug-check` requires it, so it can't hurt.